opensrp-anc/build.gradle

```diff
@@ -205,7 +208,7 @@ dependencies {
         exclude group: 'org.smartregister', module: 'android-p2p-sync'
     }
 
-    api("org.smartregister:android-p2p-sync:0.3.8-aplha4-SNAPSHOT") {
+    api("org.smartregister:android-p2p-sync:0.4.0-DEV-SNAPSHOT") {
```

Comment:
Are we using preview because p2p isn't merged yet? Please link the PR that needs merging for this to change.

Reply:
https://github.com/opensrp/android-p2p-sync/pull/85